bicep/appServiceFunction.bicep

```diff
@@ -1,15 +1,16 @@
 @description('The name of the function app that you wish to create.')
 param name string
+param kvUri string
```

Comment:
You shouldn't need to pass in the kvUri. If we can we always want to get the resource by name and then get its properties.